.cspell.json

```diff
@@ -30,7 +30,9 @@
     "ESRA",
     "pino",
     "Dockerised",
-    "JSESSIONID"
+    "JSESSIONID",
+    "probabilityof",
```

Comment:
We have a response field called `ProbabilityofDefault`, which feels like a typo. However that's what it is also called in the ACBS response so I didn't change it - should we?

Reply:
It does feel like a typo. But let's leave it as it is for now.